decentralizediso20022/src/main/resources/application-external.properties

```diff
@@ -0,0 +1,16 @@
+server.port=8443
+server.servlet.context-path=/refdata
```

Comment:
Why is the name refdata?

Reply:
I'm not sure, it was used in the WAPI. 😅

Reply:
I think you have misunderstood, that's the name of the BICS example's service in WAPI. I think we could have just `/` for now.

Reply:
I see, then how about changing it to `/api` in case there are other resources on the host?

Reply:
I think this might be a discussion we will need to have again in the future, but I do think `api` is better than `refdata`.

Reply:
Changed it to `/api`